docs/narr/assets.rst

```diff
@@ -376,14 +375,18 @@ assets by passing the optional argument, ``cachebust`` to
    from pyramid.static import QueryStringConstantCacheBuster
 
    # config is an instance of pyramid.config.Configurator
-   config.add_static_view(
-       name='static', path='mypackage:folder/static',
-       cachebust=QueryStringConstantCacheBuster(str(int(time.time()))),
-   )
+   config.add_static_view(name='static', path='mypackage:folder/static/')
+   config.add_cache_buster(
+       'mypackage:folder/static/',
+       QueryStringConstantCacheBuster(str(int(time.time()))))
+
+.. note::
+   The trailing slash on the ``add_cache_buster`` call is important to
+   indicate that it is overriding every asset in the folder and not just a
+   file named ``static``.
```

Comment:
No we don't allow it for a single file. You reminded me I need to update the docs. Thanks. Maybe just look at the api for now.